x/liquidstake/types/params.go

```diff
@@ -39,10 +39,15 @@ var (
 	AutocompoundTrigger = math.LegacyNewDecWithPrec(1, 3) // "0.001000000000000000"
 
 	// LiquidStakeProxyAcc is a proxy reserve account for delegation and undelegation.
-	LiquidStakeProxyAcc = authtypes.NewModuleAddress(ModuleName + "-LiquidStakeProxyAcc")
+	//
+	// Important: derive this address using module.Address to obtain a 32-byte version distinguishable for LSM
```

Comment:
Isn't the address already derived ? `sdk.AccAddress` returns a 32-byte address

Reply:
sdk.AccAddress is not a function, it's a typecast from `[]byte`

```go
len(sdk.AccAddress(address.Module("lsm", []byte("blah!")))) == 32
len(authtypes.NewModuleAddress(ModuleName + "-LiquidStakeProxyAcc")) == 20
```

Reply:
So turns out that `address.Module` and `authtypes.NewModuleAddress` are different things in Cosmos SDK ;)